src/onnxruntime.cc

```diff
@@ -388,221 +388,219 @@ ModelState::LoadModel(
   // to indicate that execution providers should not be added (this is
```

Comment:
Would need to update this comment to describe new behavior where we add CUDA EP for kind == AUTO.